extension.neon

```diff
@@ -88,10 +91,9 @@ services:
     - Efabrica\PHPStanLatte\VariableCollector\DynamicFilterVariables(%latte.filters%)
 
     # Error builder
-    - Efabrica\PHPStanLatte\Error\ErrorBuilder(%latte.errorPatternsToIgnore%)
+    - Efabrica\PHPStanLatte\Error\ErrorBuilder(%latte.errorPatternsToIgnore%, %latte.applicationMapping%)
```

Comment:
sem podla mna nepotrebujeme posielat ten mapping, je to trochu divne ked do error builder ide taketo nieco

Reply:
potrebuje vedet jestli je mapping nastaveny nebo ne aby vedel jesli ty chyby ignorovat